optimum/intel/openvino/modeling_base.py

```diff
@@ -114,11 +114,28 @@ def load_model(file_name: Union[str, Path]):
             file_name (`str` or `Path`):
                 The path of the model ONNX or XML file.
         """
+
+        def fix_op_names_duplicates(model: openvino.runtime.Model):
+            names = set()
+            for op in model.get_ops():
+                friendly_name = op.get_friendly_name()
+                while True:
+                    if friendly_name not in names:
+                        break
+                    friendly_name += "_"
+                names.add(friendly_name)
+                op.set_friendly_name(friendly_name)
+            return model
+
         if isinstance(file_name, str):
             file_name = Path(file_name)
         bin_file_name = file_name.with_suffix(".bin") if file_name.suffix == ".xml" else None
 
-        return core.read_model(file_name, bin_file_name)
+        model = core.read_model(file_name, bin_file_name)
+        if file_name.suffix == ".onnx":
+            model = fix_op_names_duplicates(model)  # should be called during model conversion to IR
```

Comment:
Should  `fix_op_names_duplicates` be called when converting an ONNX model to OpenVINO or is this needed to enable NNCF quantization on an OpenVINO model ?